.github/workflows/release.yml

```diff
@@ -64,8 +64,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          #- os: ubuntu-22.04
-          #  asset_name: ic-wasm-linux64
+          - os: ubuntu-22.04
+            asset_name: ic-wasm-linux64
```

Comment:
Note: I made an error in my last PR, commenting out Ubuntu 22.04 instead of 20.04.  However now both can be uncommented.